### ISSUE TITLE
When will share the loss code？ Thx

### ISSUE THREAD

**Issue by evanlin88** [shannanyinxiang/ViTEraser#2]
When will share the todo list code ："ViTEraser training code"  and "SegMIM pre-training code" , Thx

**Comment by shannanyinxiang** [comment]
We will release the remaining codes as soon as possible. Thank you for your interest in our work!
